src/test/java/com/moodysalem/TimeZoneMapperTest.java

```diff
@@ -75,25 +73,19 @@ public String getExpectedTimezone() {
     public void runTestCases() {
         final AtomicInteger succeeded = new AtomicInteger(0), failed = new AtomicInteger(0);
 
-        final Consumer<TestData> tester = (d) -> {
-            String timezone = TimezoneMapper.tzNameAt(d.getLat(), d.getLng());
-            ZoneId zone = TimezoneMapper.tzAt(d.getLat(), d.getLng());
+        for (TestData data: testData) {
+            String timezone = TimezoneMapper.tzNameAt(data.getLat(), data.getLng());
 
-            String error = String.format(EXPECT_ERROR, d.getName(), d.getLat(), d.getLng(), d.getExpectedTimezone(), timezone);
+            String error = String.format(EXPECT_ERROR, data.getName(), data.getLat(), data.getLng(),
+                    data.getExpectedTimezone(), timezone);
             try {
-                assertTrue(error, d.getExpectedTimezone().equals(timezone));
-                assertTrue(error, ZoneId.of(d.getExpectedTimezone()).equals(zone));
+                assertTrue(error, data.getExpectedTimezone().equals(timezone));
                 succeeded.incrementAndGet();
             } catch (AssertionError e) {
                 failed.incrementAndGet();
                 LOG.warning(e.getMessage());
             }
-        };
-
-        testData.forEach(tester);
-
-        // hand entered test cases
-        tester.accept(new TestData("Chicago, IL", 41.8369, -87.6847, "America/Chicago"));
```

Comment:
dang.. well in that case it's no different than the existing tests-we are still keeping track of the success percentage and catching assertion exceptions that we expect to be thrown. i kindof prefer how it was written before to explicitly check the success percentage, rather than using annotations.